scripts/dev_populate_conferences.py

```diff
@@ -435,9 +632,22 @@ def main():
     },
 }
 
+def clear_up_conf():
```

Comment:
How about `clear_conferences` to maintain parallelism with `populate_conferences`?